app/(auth)/signup.js

```diff
@@ -40,6 +41,9 @@ export default function SignUp() {
       if (password !== passwordCheck) {
         throw "Passwords don't match";
       }
+      if (!email.endsWith("@oregonstate.edu")) {
+        throw "Only @oregonstate.edu emails are allowed at this time.";
+      }
```

Comment:
I think this is good for the short term but we will want to make sure others are aware of this so when we expand to other teams we can remove this